src/EnergyPlus/DXCoils.cc

```diff
@@ -14988,31 +15071,29 @@ void GetFanIndexForTwoSpeedCoil(
     int BranchNum;
     int CompNum;
 
+    auto &thisDXCoil = state.dataDXCoils->DXCoil(CoolingCoilIndex);
+
     FoundBranch = 0;
     FoundAirSysNum = 0;
     SupplyFanIndex = 0;
     SupplyFanName = "n/a";
     for (AirSysNum = 1; AirSysNum <= NumPrimaryAirSys; ++AirSysNum) {
```

Comment:
While you are at it, may want to downscope the loop induction variables.

Reply:
Done.